docs/src/main/asciidoc/reactive-messaging.adoc

```diff
@@ -272,30 +274,34 @@ Then, creates a new JAX-RS resource with the following content:
 ----
 package org.acme.vertx;
 
-import io.vertx.axle.core.eventbus.EventBus;
-import io.vertx.axle.core.eventbus.Message;
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.eventbus.EventBus;
+import io.vertx.mutiny.core.eventbus.Message;
+import org.jboss.resteasy.annotations.jaxrs.PathParam;
 
 import javax.inject.Inject;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import java.util.concurrent.CompletionStage;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
 
-@Path("/hello")
+@Path("/async")
 public class EventResource {
 
-    @Inject EventBus bus;
+    @Inject
+    EventBus bus;
 
     @GET
-    @Path("/async/{name}")
-    public CompletionStage<String> hello(@PathParam("name") String name) {
-        return bus.<String>request("greeting", name)                        // <1>
-                .thenApply(Message::body);                                  // <2>
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("{name}")
+    public Uni<String> greeting(@PathParam String name) {
+        return bus.<String>request("greeting", name)
+                .onItem().apply(Message::body);
     }
 }
 ----
-<1> send the `name` to the `greeting` address
-<2> when we get the reply, extract the body and send this as response to the user
+<1> send the `name` to the `greeting` address and request a response
+<2> when we get the response, extract the body and send it to the user
```

Comment:
Same here: the callouts were removed from the code.

Reply:
Fixed.